cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java

```diff
@@ -89,7 +89,7 @@ public ApplicationsOperations(DefaultCloudFoundryOperations cloudFoundryOperatio
 
         return this.cloudFoundryOperations.applications()
                 .delete(request)
-                .doOnSuccess(aVoid -> log.info("Application " + applicationName + " has been successfully removed."));
+                .doOnSuccess(aVoid -> log.info("App removed: ",applicationName));
```

Comment:
log.info("App removed: ", applicationName));

with a space.

Reply:
I think it is not necessary, because an app is only under 1 space. I also changed it to be identical to the create method with log.info ().